app/Models/AssetMaintenance.php

```diff
@@ -94,8 +94,8 @@ public function setIsWarrantyAttribute($value)
      */
     public function setCostAttribute($value)
     {
-        $value = Helper::ParseFloat($value);
-        if ($value == '0.0') {
+        $value = Helper::ParseCurrency($value);
+        if ($value === 0) {
```

Comment:
I’m wondering if `ParseCurrency` is ever going to return an **integer** with a value of 0 - I’m assuming it’d probably return a **float** with a value of 0. So my gut here is that a double-equals would be better.

Reply:
You are right. I changed the comparison to two equal signs now.